packages/core/admin/admin/src/pages/AuthPage/components/Login/BaseLogin.js

```diff
@@ -1,84 +1,99 @@
 import React from 'react';
-import { Checkbox } from '@buffetjs/core';
-import { useIntl } from 'react-intl';
-import { get } from 'lodash';
+import { Box, Stack, H1, Text, Subtitle, Button, Checkbox, TextInput, Main } from '@strapi/parts';
 import PropTypes from 'prop-types';
-import { BaselineAlignment } from '@strapi/helper-plugin';
+import styled from 'styled-components';
+import { useIntl } from 'react-intl';
+import { Formik, Form } from 'formik';
+
+import { Column } from '../../../../layouts/UnauthenticatedLayout';
+import { useConfigurations } from '../../../../hooks';
+import FormikFocusError from './FormikFocusError';
 
-import Button from '../../../../components/FullWidthButton';
-import AuthLink from '../AuthLink';
-import Box from '../Box';
-import Input from '../Input';
-import Logo from '../Logo';
-import Section from '../Section';
+const AuthButton = styled(Button)`
+  display: inline-block;
+  width: 100%;
+`;
 
-const Login = ({ children, formErrors, modifiedData, onChange, onSubmit, requestError }) => {
+const Login = ({ onSubmit, schema }) => {
+  const { authLogo } = useConfigurations();
   const { formatMessage } = useIntl();
 
   return (
-    <>
-      <Section textAlign="center">
-        <Logo />
-      </Section>
-      <Section withBackground>
-        <BaselineAlignment top size="25px">
-          <Box errorMessage={get(requestError, 'errorMessage', null)}>
-            <form onSubmit={onSubmit}>
-              <Input
-                autoFocus
-                error={formErrors.email}
-                label="Auth.form.email.label"
+    <Main labelledBy="welcome">
+      <Formik
+        enableReinitialize
+        initialValues={{
+          email: '',
+          password: '',
+        }}
+        onSubmit={onSubmit}
+        validationSchema={schema}
+        validateOnChange={false}
+      >
+        {({ values, errors, handleChange }) => (
+          <Form noValidate>
+            <FormikFocusError />
+            <Column>
+              <img src={authLogo} alt="strapi-app-logo" style={{ height: '72px' }} />
```

Comment:
The alt should be something readable by a human 😊 In this scenario, if the logo is important, I would suggest to switch `strapi-app-logo` by `Strapi`

Reply:
This is why I put a comment on our notion doc about the custom configuration. The AuthLogo is customizable. I would follow your advice about the aria-hidden on that img.

Reply:
I've removed the tag in the DS, we now have to pass an empty alt attribute instead. This is easier to manage since it's now the user-lands reponsability to announce a title (or not)